DeckCustomization/DeckCustomization.cs

```diff
@@ -289,6 +292,7 @@ void UpdateRarity(float val, CardInfo.Rarity rarity)
             {
                 RarityRarities[rarity] = val / 100f;
                 RarityRaritiesConfig[rarity].Value = val / 100f;
+                RarityLib.Utils.RarityUtils.GetRarityData(rarity).calculatedRarity = val / 100f;
```

Comment:
What's this doing?

Reply:
it is informing rarityLib of the changed rarity, the idea is that we store the rarity with rarityLib so that all someone would have to do to mess with a rarity mid game is depend on rarityLib rather then depending on DC like is happening now